dashboard/generateLayer.py

```diff
@@ -200,15 +201,15 @@ def generateLayer(dataSources, filename_output):
                         },
                     }
                 )
-                
+
                 finishedLayers.append(dataField)
             #    except:
             #        exceptions.append(dataField)
 
         # print("Processed category: ", key_category)
         o += 1
-        
-        #print(layers[key_category].keys())
+
+        # print(layers[key_category].keys())
 
     with open(filename_output, "w") as outs:
         jsonDumps = json.dumps(output, indent=4, sort_keys=True)
```

Comment:
Inane question no. 2) this is doing something similar to the first section (i.e. creating layers for the variables in the map?)

Reply:
I'll mark this as resolved as it will relate to a future PR :)